Library/PTURLRequestOperation.m

```diff
@@ -50,19 +82,22 @@ - (void)start
 
   [self setExecuting:YES];
 
+  self.sessionDelegate = [PTURLRequestOperationURLSessionDelegate new];
+  self.sessionDelegate.delegate = self;
   NSURLSessionConfiguration *sessionConfig = [NSURLSessionConfiguration defaultSessionConfiguration];
-  URLSession = [NSURLSession sessionWithConfiguration:sessionConfig delegate:self delegateQueue:nil];
-  NSURLSessionDataTask *task = [URLSession dataTaskWithRequest:URLRequest];
+  _URLSession = [NSURLSession sessionWithConfiguration:sessionConfig delegate:self.sessionDelegate delegateQueue:nil];
```

Comment:
`NSURLSession` holds a `strong` reference to the `delegate`. So I made a proxy so `self` and possibly the rest of pusher isn't captured.